tasks/omnibus.py

```diff
@@ -94,6 +97,20 @@ def get_omnibus_env(
 ):
     env = load_dependencies(ctx)
 
+    # Discard windows variables when not on Windows
+    if sys.platform != 'win32':
+        windows_only_vars = [
+            'WINDOWS_DDNPM_DRIVER',
+            'WINDOWS_DDNPM_VERSION',
+            'WINDOWS_DDNPM_SHASUM',
+            'WINDOWS_DDPROCMON_DRIVER',
+            'WINDOWS_DDPROCMON_VERSION',
+            'WINDOWS_DDPROCMON_SHASUM',
+        ]
+        for var in windows_only_vars:
+            if var in env:
+                del env[var]
```

Comment:
```suggestion
            env.pop(var, None)
```

Reply:
🤷 I considered that, but it felt off to me to use `pop` and not using the value, but I'm fine changing it.

Reply:
linters may indeed complain we're not using the value - was only a suggestion.